apps/web-giddh/src/app/daybook/export-daybook/export-daybook.component.html

```diff
@@ -3,97 +3,46 @@
-                        type="radio"
-                        [(ngModel)]="fileType"
-                        [value]="'pdf'"
-                    />
-                    <label for="pdf" class="radio-custom-label">{{ commonLocaleData?.app_pdf }}</label>
-                </span>
-
-                <span class="p-0 mr-r3">
-                    <input
-                        id="excel"
-                        class="radio-custom cp ml-0"
-                        name="fileType"
-                        type="radio"
-                        [(ngModel)]="fileType"
-                        [value]="'xlsx'"
-                    />
-                    <label for="excel" class="radio-custom-label">{{ commonLocaleData?.app_excel }}</label>
-                </span>
+                <mat-radio-group color="primary" name="fileType" [(ngModel)]="fileType">
+                    <mat-radio-button id="pdf" [value]="'pdf'">{{ commonLocaleData?.app_pdf }}</mat-radio-button>
+                    <mat-radio-button id="excel" [value]="'xlsx'">{{ commonLocaleData?.app_excel }}</mat-radio-button>
+                </mat-radio-group>
             </div>
             <div class="pd-b2 clearfix">
                 <label class="w100 d-sm-inline-flex d-flex">{{ localeData?.order }} </label>
-
-                <span class="p-0 mr-r3">
-                    <input
-                        id="ascending"
-                        class="radio-custom cp ml-0"
-                        name="order"
-                        [(ngModel)]="order"
-                        type="radio"
-                        [value]="'asc'"
-                    />
-                    <label for="ascending" class="radio-custom-label">{{ commonLocaleData?.app_ascending }}</label>
-                </span>
-
-                <span class="p-0 mr-r3">
-                    <input
-                        id="descending"
-                        class="radio-custom cp ml-0"
-                        name="order"
-                        [(ngModel)]="order"
-                        type="radio"
-                        [value]="'desc'"
-                    />
-                    <label for="descending" class="radio-custom-label">{{ commonLocaleData?.app_descending }}</label>
-                </span>
+                <mat-radio-group color="primary" name="order" [(ngModel)]="order">
+                    <mat-radio-button id="ascending" [value]="'asc'"
+                        >{{ commonLocaleData?.app_ascending }}
+                    </mat-radio-button>
+                    <mat-radio-button id="descending" [value]="'desc'"
+                        >{{ commonLocaleData?.app_descending }}
+                    </mat-radio-button>
+                </mat-radio-group>
             </div>
-            <button class="btn-success btn" (click)="exportLedger()">{{ commonLocaleData?.app_download }}</button>
+            <button mat-raised-button color="primary" (click)="exportLedger()">
```

Comment:
@ravinderwalkover remove the color primary and add the class which are used above @ravinderwalkover

Reply:
Done